services/mock-service/src/fsp-integration/safaricom/safaricom.mock.service.ts

```diff
@@ -133,21 +139,39 @@ export class SafaricomMockService {
         },
       },
     };
+    const timeoutCallbackResponse = {
+      InitiatorName: transferDto.InitiatorName,
+      SecurityCredential: transferDto.SecurityCredential,
+      CommandID: transferDto.CommandID,
+      Amount: transferDto.Amount,
+      PartyA: transferDto.PartyA,
+      PartyB: transferDto.PartyB,
+      Remarks: transferDto.Remarks,
+      QueueTimeOutURL: transferDto.QueueTimeOutURL,
+      ResultURL: transferDto.ResultURL,
+      OriginatorConversationID: transferDto.OriginatorConversationID,
+      IDType: transferDto.IDType,
+      IDNumber: transferDto.IDNumber,
+    };
+
+    const httpService = new HttpService();
 
     // Switch between mock scenarios
-    let Status;
+    let response = {};
+    let url = `${EXTERNAL_API_ROOT}/${API_PATHS.safaricomCallback}`;
     if (mockScenario === MockScenario.success) {
-      Status = successStatus;
+      response = {
+        Result: successStatus.Result,
+      };
     } else if (mockScenario === MockScenario.errorOnCallback) {
-      Status = otherFailureStatus;
+      response = {
+        Result: otherFailureStatus.Result,
```

Comment:
can you change this name 'otherFailureStatus', as this used to align with a mock-scenario-name, but no longer.